recipe/meta.yaml

```
@@ -1,45 +1,75 @@
 {% set name = "langchain-community" %}
-{% set version = "0.3.12" %}
+{% set version = "0.3.30" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain_community-{{ version }}.tar.gz
-  sha256: b4694f34c7214dede03fe5a75e9f335e16bd788dfa6ca279302ad357bf0d0fc4
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: df68fbde7f7fa5142ab93b0cbc104916b12ab4163e200edd933ee93e67956ee9
 
 build:
-  skip: true  # [py<39 or s390x]
+  skip: true  # [py<39]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
     - python
-    - poetry-core >=1.0.0
     - pip
+    - pdm-backend
   run:
     - python
-    - langchain <0.4.0,>=0.3.12
-    - langchain-core <0.4.0,>=0.3.25
-    - sqlalchemy <3,>=1.4
-    - requests >=2.0.0,<3.0.0
-    - pyyaml >=5.3
-    - numpy >=1.22.4,<2.0.0  # [py<312]
-    - numpy >=1.26.2,<3.0.0  # [py>=312]
+    - langchain >=0.3.27,<2.0.0
```

https://inspector.pypi.io/project/langchain-community/0.3.30/packages/d7/32/852facdba14140bbfc9b02e6dcb00fe2e0c5f50901d512a473351cf013e2/langchain_community-0.3.30.tar.gz/langchain_community-0.3.30/pyproject.toml#line.11
```suggestion
    - langchain >=0.3.75,<2.0.0
```